Tests/test_map.gd

```diff
@@ -0,0 +1,7 @@
+extends GutTest ## Tests for MapManager, more will be added in the future
+
+var test_generator = load("res://Map/MapManager.gd")
+
+func test_map_gen():
+	var result = test_generator.create_map()
+	assert_eq(result, result, "Expected to return generated class")
```

Comment:
A better test here would be `assert_not_null(result, "Expected to return generated class")`. We would want to be checking for equality if we make a test object within this file to compare it against!